README.md

```diff
@@ -1 +1,10 @@
-# etch-a-sketch_Obaid
\ No newline at end of file
+# etch a sketch Obaid Project
+
+#WHY 
+These files here are project for practising what we learned in Gaza Sky Geeks Web Programing Fundemental Course
+
+#WHAT 
+this project is to make etch a sketch so as to be able to draw on the page by moving mosue over, painting by random color/colors or py selecting a color
+
+#HOW 
+this project created by HTML CSS and Javascript, as i created divs dynamicly using javascript and when the mose hover the color of the divs change and we can select by which colors we can paint. 
```

Comment:
this is good :ok_hand:  but here what we meant with "How" is from a user perspective which is the user journey.
for instance:
>  as a user, I would be able to erase the colored grids.

whenever you add a new feature, you may add its own user journey.
PS: make it only one sentence each, specific and simple.

Reply:
ok